packages/cypress/src/plugins/preprocessor-vite.ts

```diff
@@ -40,7 +40,9 @@ const cache = new Map<string, string>();
  * This preprocessor shouldn't be used directly.
  * Instead, use the nxE2EPreset(__filename, { bundler: 'vite' }) function instead.
  */
-function vitePreprocessor(userConfigPath?: string): CypressPreprocessor {
+function vitePreprocessor(
+  configOverrides: InlineConfig = {}
```

Comment:
Usages of `vitePreprocessor('/path/to/vite.config.ts')` will break right? Do we need to overload the function or deprecate something?

Reply:
Nevermind, it's an internal API. :)